src/ApiCalls.js

```diff
@@ -1,8 +1,14 @@
 const apiCalls = {
-
   fetchData() {
-    return fetch("https://rancid-tomatillos.herokuapp.com/api/v2/movies").then(response => response.json())
+    return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies').then(
+      response => response.json()
+    );
   },
 
-}
+  fetchOneMovie(id) {
+    return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`).then(
+      response => response.json()
+    );
+  }
+};
```

Comment:
These fetches look very similar - maybe we could refactor so that we have one fetch that takes in the entire url as an argument? Additionally, make sure to add some `.catch`'s too!